src/interactivity/onecore/ConIoSrvComm.cpp

```diff
@@ -359,22 +348,22 @@ VOID ConIoSrvComm::HandleFocusEvent(PCIS_EVENT Event)
             // This is necessary because the engine cannot be allowed to
             // request ownership of the display before whatever instance
             // of conhost was using it before has relinquished it.
-            if (!pWddmConEngine->IsInitialized())
+            if (!wddmConEngine.IsInitialized())
             {
-                hr = pWddmConEngine->Initialize();
+                hr = wddmConEngine.Initialize();
                 LOG_IF_FAILED(hr);
 
                 // Right after we initialize, synchronize the screen/viewport states with the WddmCon surface dimensions
                 if (SUCCEEDED(hr))
                 {
-                    const RECT rcOld = { 0 };
+                    const RECT rcOld{};

```

Comment:
AuditMode complained that the Y member wasn't initialized for the `COORD` below (kinda weird?), so I went ahead and C++ified all those initializers in this project.